_includes/use-cases/use-case-consumption-query-attributes.md

```diff
@@ -0,0 +1,22 @@
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case is intended to query all [LocalAttributes]({% link _docs_integrate/data-model-overview.md %}#localattribute)
+related to the identity.
```

Comment:
> related to the identity.
I guess this is wrong here. The use-case can be used to query ANY LocalAttribute, which could also be an Attribute shared to a peer or a received IdentityAttribute from a peer.